literature/literature.bib

```diff
@@ -12673,6 +13372,20 @@ @article{SFG+:SCP14
 	tttags    = {Feature Orientation,SPL Analysis,Modularity},
 }
 
+@article{CB:SEN14,
+	author    = {Castelluccia, Daniela and Boffoli, Nicola},
+	title     = {{Service-Oriented Product Lines: A Systematic Mapping Studys}},
+	journal   = sen,
+	volume    = {39},
+	number    = {2},
+	pages     = {1--6},
+	doi       = {10.1145/2579281.2579294},
+	publisher = acm,
+	year      = 2014,
+	keywords  = {Service-oriented Architecture, Software product line, Mapping study},
```

Comment:
Also lower case? if this is not consistent throughout the document already, then you do not have to bother adapting it. Then an issue reporting this inconsistency would be great.